README.rst

```diff
@@ -88,7 +89,10 @@ Before running tests, please ensure that following environment variables are set
     ==> BASIC_AUTH_PASSWORD
     ==> USER_LOGIN_EMAIL
     ==> USER_LOGIN_PASSWORD
-
+    ==> COURSE_RUN
+    ==> COURSE_DISPLAY_NAME
+    ==> COURSE_NUMBER
+    ==> COURSE_ORG
 
 To run all the tests:
 
```

Comment:
@aisha-sajid concerning the flaky tests, I would not put it into the documentation. We do not do it for the edx-platform. Plus, we are in talks concerning removing the flaky test library in the near future.